src/python/pants/option/config_test.py

```diff
@@ -1,17 +1,33 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-import textwrap
-import unittest
+from textwrap import dedent
+from typing import Dict
+
+from twitter.common.collections import OrderedSet
 
 from pants.option.config import Config
-from pants.util.contextutil import temporary_file, temporary_file_path
+from pants.testutil.test_base import TestBase
+from pants.util.contextutil import temporary_file
+
 
+class ConfigTest(TestBase):
 
-class ConfigTest(unittest.TestCase):
+  def _setup_config(self, config1_content: str, config2_content: str, *, suffix: str) -> Config:
+    with temporary_file(binary_mode=False, suffix=suffix) as config1, \
+      temporary_file(binary_mode=False, suffix=suffix) as config2:
+      config1.write(config1_content)
+      config1.close()
+      config2.write(config2_content)
+      config2.close()
+      parsed_config = Config.load(
+        config_paths=[config1.name, config2.name], seed_values={"buildroot": self.build_root}
```

Comment:
Originally, we used `temporary_file_path` to generate the build root. Now, we use `TestBase.build_root`.